pipeline/08-api.R

```diff
@@ -10,9 +10,7 @@ purrr::walk(list.files("R/", "\\.R$", full.names = TRUE), source)
 
 # Load additional dev R libraries (see README#managing-r-dependencies)
 suppressPackageStartupMessages({
-  library(DBI)
```

Comment:
I'm pretty sure `DBI` and `RJDBC` are not being used by this pipeline stage, but perhaps I'm missing something that's not obvious in the code.